docs/UserGuide.adoc

```diff
@@ -142,17 +188,12 @@ Examples:
 
 * `removetag 2 \t hawker`
 
-==== Listing all eateries: `list`
-
-Shows a list of all eateries in the app. +
-Format: `list`
-
-==== Getting a random eatery: `random`
+===== Getting a random eatery: `random`
 
 Shows a completely random restaurant - Useful in case you cannot decide what to eat! +
 Format: `random`
 
-==== Viewing an eatery: `show`
+===== Viewing an eatery: `show`
 
 Shows an eatery with all its details - Address, category, tags, reviews, external reviews and a map of its location. +
```

Comment:
Shows an eatery with all its details i.e. address, category, tags, reviews, external reviews (from Google), and a map of its location.